src/.nuget/optdata/ibcmerge.csproj

```diff
@@ -9,8 +9,20 @@
     <PackageReference Include="microsoft.dotnet.ibcmerge" Version="[$(IbcMergePackageVersion)]" Condition="'$(IbcMergePackageVersion)'!=''" />
   </ItemGroup>
 
+  <!--                                                                       -->
+  <!-- Task: DumpIbcMergePackageVersion                                       -->
+  <!--                                                                       -->
+  <!-- Notes:                                                                -->
+  <!--                                                                       -->
+  <!-- DumpIbcMergePackageVersion is used by build.sh and build.cmd to pass  -->
+  <!-- the version information to cmake. The task will write a file to the   -->
+  <!-- intermediates directory to be read back by build.cmd/sh. The path for -->
+  <!-- the file is $(__IntermediatesDir)\ibcmergeversion.txt      -->
+  <!--                                                                       -->
   <Target Name="DumpIbcMergePackageVersion">
+    <!-- Output the package version to stdout for backcompat -->
```

Comment:
What is the backcompat concern? Are there scenarios where we use older build scripts with newer projects?
I like keeping the output message in general - just confused by the comment about backcompat.

Reply:
build.sh/build.cmd are not the only scripts which rely on the std out having the version number.

Reply:
Found that the other script that was referencing the projects has bit rotted and is no longer in use.